### ISSUE TITLE
Sprites and textures not appearing??

### ISSUE THREAD

**Issue by ShadowMan44** [sirjuddington/SLADE#1404]
**SLADE Version**: 3.2.1.
**Operating System**: Windows 10

**Issue Details**:
I just installed the latest version on my laptop and the graphics on the right side of the screen are completely broken. I installed a much older version of the program and opened the same pk3 and they worked just fine. I also tried using the previous build and the same issue occurred.

![unknown](https://user-images.githubusercontent.com/42453370/178906948-5c859999-9e6c-4b11-9a8e-d1a8c08052fc.png)

I have not seen anybody else bring this issue up at all so I figured I'd go ahead and complain about this. My best guess is that it has something to do with the structure of the new _and quite frankly, extremely unnecessary_ interface change.

**Comment by Gaerzi** [comment]
Open the console and look at any error messages. I doubt it's the UI change, more likely the change in how OpenGL is wrapped up.

**Comment by ParadoX6891** [comment]
![изображение](https://user-images.githubusercontent.com/23018572/190641992-5a0648d5-f107-47ad-a808-7a0411355a62.png)
I have analogical problem. 3.1.13 working perfectly.

**Comment by sirjuddington** [comment]
Yeah this one is a system/driver issue, OpenGL can't be initialized correctly so it doesn't support textures with non-power-of-two dimensions. Old versions of SLADE had some hacks to get around this limitation but it was overcomplicating the texture code and wouldn't work in the map editor so was removed.

Hopefully once I implement modern opengl support issues like this won't happen, since I think some drivers don't like opengl 2.x (which SLADE currently uses) and just revert to 1.1 for whatever reason

**Comment by ShadowMan44** [comment]
> Yeah this one is a system/driver issue, OpenGL can't be initialized correctly so it doesn't support textures with non-power-of-two dimensions. Old versions of SLADE had some hacks to get around this limitation but it was overcomplicating the texture code and wouldn't work in the map editor so was removed.
> 
> Hopefully once I implement modern opengl support issues like this won't happen, since I think some drivers don't like opengl 2.x (which SLADE currently uses) and just revert to 1.1 for whatever reason

How soon do you think you will be able to drop the next Slade release?
